src/base/server_type.cpp

```diff
@@ -105,23 +105,24 @@ void set_server_type (SERVER_TYPE type)
 int init_server_type (const char *db_name)
 {
   int er_code = NO_ERROR;
-  server_type_config parameter_value = (server_type_config) prm_get_integer_value (PRM_ID_SERVER_TYPE);
-  g_transaction_server_type = get_value_from_config ((transaction_server_type_config) prm_get_integer_value (
-				      PRM_ID_TRANSACTION_SERVER_TYPE));
+  const auto server_type_from_config = (server_type_config) prm_get_integer_value (PRM_ID_SERVER_TYPE);
+  g_transaction_server_type = get_transaction_server_type_from_config (
+				      (transaction_server_type_config) prm_get_integer_value (
+					  PRM_ID_TRANSACTION_SERVER_TYPE));
```

Comment:
Split this into two statements:
```suggestion
  const auto transaction_server_type_from_config = (transaction_server_type_config) prm_get_integer_value (PRM_ID_TRANSACTION_SERVER_TYPE));
  g_transaction_server_type = get_transaction_server_type_from_config (transaction_server_type_from_config);
```